src/server/services/zorgned/zorgned-helpers.ts

```diff
@@ -19,32 +20,36 @@ export function parseLabelContent(
   return rText;
 }
 
-export function hasFutureDate(dateStr: string | null, compareDate: Date) {
+export function isFutureDate(dateStr: string | null, compareDate: Date) {
   if (!dateStr) {
     return false;
   }
-  return !isDateInPast(dateStr, compareDate);
+  return isSameDay(parseISO(dateStr), compareDate)
+    ? false
+    : !isDateInPast(dateStr, compareDate);
 }
 
-export function hasHistoricDate(dateStr: string | null, compareDate: Date) {
+export function isHistoricDate(dateStr: string | null, compareDate: Date) {
```

Comment:
Mooi de naam verbeterd, maar ik vroeg me nog wel af wat historic is? Dus na het lezen van de implementatie kom ik erachter dat het `true` is als de datum ergens voor gisteren zich afspeelt dus ik denk dat iets als `isBeforeYesterday` misschien beter vertelt waar het over gaat.

En een comment waarom `null` false teruggeeft is ook wel fijn, maar een `null` is ook geen datum dus dat is ook geen historic date, dus zo snap ik het ook wel. Maar als je er wat in ziet, is het een idee.

Reply:
het is `isBeforeToday`

Reply:
Ah inderdaad, mijn fout maar nice ik zie het, thanks